org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/AbstractMatcher.java

```diff
@@ -17,6 +17,7 @@
 import org.objectweb.asm.Opcodes;
 import org.objectweb.asm.tree.AbstractInsnNode;
 import org.objectweb.asm.tree.MethodInsnNode;
+import org.objectweb.asm.tree.MethodNode;
 import org.objectweb.asm.tree.VarInsnNode;
 
 abstract class AbstractMatcher {
```

Comment:
I wonder whether it would be worth to add unit tests for AbstractMatcher. Especially as we miss several branches with the rest of our test suite.

Reply:
@marchof Let's do this separately.

Reply:
👍